test/Microsoft.DocAsCode.Build.Engine.Tests/DocumentProcessors/YamlDocumentProcessor.cs

```diff
@@ -175,7 +175,7 @@ protected virtual JsonSerializer GetSerializer()
                 {
                     new Newtonsoft.Json.Converters.StringEnumConverter(),
                 },
-                TypeNameHandling = TypeNameHandling.All,
+                TypeNameHandling = TypeNameHandling.All, // lgtm [cs/unsafe-type-name-handling]
```

Comment:
Should this have a comment about why it's safe? Like, is the input data part of this repository and not controllable by adversaries?

Reply:
this is a test class. input data is test data.